docs/modules/commons/pages/expressions.adoc

```diff
@@ -294,6 +294,27 @@ Compress the input string to GZip and encode compressed bytes to Base64 format
 |`H4sIAAAAAAAAACvLLMtMKS0GANIHCdkHAAAA`
 |===
 
+
+=== `hash`
+
+Calculates the hash using specific hashing algorithm
+
+[source, subs="+quotes"]
+----
+#{hash($input, $algorithm)}
+----
+
+* *`$input`* - any string for which need to get the hash
+* *`$algorithm`* - hash algorithm (can use MD2, MD5, SHA-1, SHA-256)
```

Comment:
it makese sense to add SHA-384 and SHA-512 as well

Reply:
also add link to corresponding rfc like https://datatracker.ietf.org/doc/html/rfc1319[MD2] ?